components/hab/habitat/plan.sh

```diff
@@ -45,6 +45,7 @@ _common_prepare() {
 }
 
 pkg_version() {
+  echo "THING THING THING: $BUILDKITE_BUILD_ID"
```

Comment:
since this is a studio, do you need to inject it as `HAB_STUDIO_SECRET_BUILDKITE_BUILD_ID`?

Reply:
"THING THING THING", indeed.